acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -70,6 +74,7 @@ test:
     - odoo-{{{ odoo.series }}}
   script:
     - install_acsoo
+    - start_ssh_agent  # for pushing translations
```

Comment:
Maybe we could call ```start_ssh_agent``` in the ```makepot``` function?

Reply:
The problem I see with that is it hides `start_ssh_agent` a little bit outside of the main code path, and therefore there is the risk to start it twice. We could improve start_ssh_agent to make it reentrant safe, though.